mapit_gb/management/commands/mapit_UK_add_ons_to_gss.py

```diff
@@ -30,21 +31,29 @@ class Command(NoArgsCommand):
     help = 'Inserts the old ONS codes into mapit'
 
     def handle_noargs(self, **options):
-        code_changes = codecs.open(os.path.join(
+        python_version = sys.version_info[0]
+
+        code_change_file_path = os.path.join(
             os.path.dirname(__file__),
-            '../../data/BL-2010-10-code-change.csv'
-            ), 'r', 'latin-1')
-        mapping = csv.reader(code_changes)
+            '../../data/BL-2010-10-code-change.csv')
+        if python_version < 3:
+            mapping = csv.reader(open(code_change_file_path))
+        else:
+            code_changes = codecs.open(code_change_file_path, 'r', 'latin-1')
+            mapping = csv.reader(code_changes)
         next(mapping)
         for row in mapping:
             new_code, name, old_code = row[0], row[1], row[3]
```

Comment:
Since we know now that the `name` field will probably have encoding errors under Python 2 (and we aren't using it anyway) it would be good to not assign that field to a variable at all, and explain why in the commit message.